src/common.rs

```diff
@@ -866,22 +921,26 @@ impl From<&String> for Identifier {
 pub enum IdentifierRef<'a> {
     /// This variant is case sensitive
     /// "fOo""bAr""" is stored as fOo"bAr"
-    Quoted(&'a str),
+    Quoted(&'a IdentifierWithSpan),
```

Comment:
`IdentifierRef` should be left as is with just a `&'a str`.

`IdentifierRef` is not included in the AST anywhere.
Instead the IdentifierRef is used only for comparing against a regular `Identifier` without allocating.
Therefore the IdentifierRef will never have a meaningful span.

Reply:
ok